Gemfile

```
@@ -16,6 +16,7 @@ gem "webpacker", "~> 5.0"
 # Reduces boot times through caching; required in config/boot.rb
 gem "bootsnap", ">= 1.4.2", require: false
 
+gem "view_component", path: ENV["VIEW_COMPONENT_PATH"] if ENV["VIEW_COMPONENT_PATH"]
```

```suggestion
gem "view_component", github: "github/view_component", branch: "main"
```

Would this work while also allowing https://bundler.io/man/bundle-config.1.html#LOCAL-GIT-REPOS to work?

---

were we getting view_components for free from Rails before this change?

---

I think it may have been coming from the `gemspec` before this addition.

---

I'm not sure that we want to **always** override the `view_component` defined in `.gemspec`
I don't know what the implications would be